Python/Product/TestAdapter.Executor/Pytest/PytestTestDiscoverer.cs

```diff
@@ -149,7 +149,12 @@ ITestCaseDiscoverySink discoverySink
             //Note pytest specific arguments go after this separator
             arguments.Add("--");
             arguments.Add("--cache-clear");
-            arguments.Add(String.Format("--rootdir={0}", projSettings.ProjectHome));
```

Comment:
maybe both
```c#
arguments.Add(String.Format("--rootdir={0}", projSettings.ProjectHome));
arguments.Add(String.Format("{0}\\{1}", projSettings.ProjectHome, projSettings.UnitTestRootDir));
```

Reply:
@bschnurr @AdamYoblick I updated the code again, and added a few more checks as you suggested. Could you take a look again? Thanks!

Reply:
Looks good to me 😄